opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/ServerTracingAutoConfiguration.java

```diff
@@ -25,11 +29,26 @@
 public class ServerTracingAutoConfiguration {
     private static final Logger log = Logger.getLogger(ServerTracingAutoConfiguration.class.getName());
 
+    @Autowired(required=false)
+    @Qualifier("opentracing.http.skipPattern")
+    private Set<String> skipPatterns;
```

Comment:
I guess 3rd artifacts would never provide `SkipPattern` bean and would delegate this configuration to the integration code (user). So if we introduce this it will be used only by our integrations (e.g. prometheus-metrics PR). As a developer of a larger app I would maybe prefer to have one skip pattern and control changes on it.

Could we somehow alter `WebTracingConfiguration` and change SkipPattern in other integrations via BeanPostProcessing etc?

Although, I'm not sure what is best. But right now there are two ways how to define skip pattern which is not ideal. Or we could keep this code as private and inject the set on servlet filter and do union with configuration class.